st2common/st2common/services/datastore.py

```diff
@@ -61,7 +61,7 @@ def list_values(self, local=True, prefix=None):
         kvps = client.keys.get_all(prefix=key_prefix)
         return kvps
 
-    def get_value(self, name, local=True, scope=SYSTEM_SCOPE, decrypt=False):
+    def get_value(self, name, local=True, scope=FULL_SYSTEM_SCOPE, decrypt=False):
```

Comment:
Same here and in many other modified files - valid values for "scope" option should still be "user" and "system".

I assume you just used sed to replace those occurrences in all the files or similar, but I don't think we should replace the actual valid values for "scope" option.

One thing is the template notation (e.g. "{{st2kv.user.foo}}") and the other thing is valid valued for "scope" (e.g. "user") option.